src/lib/chat-ui.js

```diff
@@ -541,11 +552,24 @@ const ChatUi = {
     }
   },
 
-  /**
-   * adds new message to lastQuestionData.message, clears the input field and visualizes it
-   *
-   * @returns {void}
-   */
+  async loadInitialMessageSettings() {
+    let actionCodes = actionService.getActionCodes(
+      this.assistant.initialMessage.content,
+      actionService.ACTION_CODE_REGEX,
+    );
+
+    let popUpHtml = getPopUp(actionCodes[0]);
+
+    if (popUpHtml) {
+      this.elements.chatbotContainer.appendChild(popUpHtml);
+      setTimeout(() => {
```

Comment:
@tuseto 
Can't you just use the delay function and omit the setTimeout?

Reply:
I can, but I don't want to. I even want to remove the delay function.